src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -54,7 +54,7 @@ public static bool TryWriteBytes(Span<byte> destination, bool value)
         public static byte[] GetBytes(char value)
         {
             byte[] bytes = new byte[sizeof(char)];
-            Unsafe.As<byte, char>(ref bytes[0]) = value;
+            MemoryMarshal.Write(bytes, value);
```

Comment:
```suggestion
            success = TryWriteBytes(bytes, value);
            Debug.Assert(success);
```

```suggestion
            _ = TryWriteBytes(bytes, value);
```

@EgorBo

Reply:
yeah I guess so, but can you apply it everywhere?